docs/storage.rst

```diff
@@ -146,210 +146,250 @@ start to diminish.
 
-.. code-block:: console
+            sudo nano /etc/postgresql/<version>/main/pg_hba.conf
 
-    sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+      - Add the following entry at the end of the file:
 
-Check whether PostgreSQL 13 is available using the command shown below
+         .. code-block:: ini
 
-.. code-block:: console
+            host    all    all    0.0.0.0/0    trust
 
-    sudo yum search -y postgresql13
+         Each entry in the file consists of five fields, these fields are described below:
 
-Once you have confirmed that PostgreSQL 13 repositories are available on your system. Then, you can proceed to install PostgreSQL 13
+            .. list-table::
+               :widths: 25 55 20
+               :header-rows: 1
 
-.. code-block:: console
+               * - Field
+                 - Description
+                 - Example
+               * - Connection
+                 - This specifies that the rule applies to TCP/IP connections.
+                 - host
+               * - Database
+                 - The name of the database to which the rule is applied. The reserve name *all* applies the rule to all databases.
+                 - all
+               * - User
+                 - The user to which the rule applies. The reserved username of *all* may be used to apply the rule to all users.
+                 - all
+               * - Address
+                 - The network mask to allow connections from. The example shown allows connections from all IPv4 addresses. Replace this with a specific network mask for better security (e.g. `192.168.1.0/24` limits connections to just those from the network 192.168.1.xxx). Either IPV4 or IPV6 network masks may be supplied.
+                 - 0.0.0.0/0
+               * - Authentication Method
+                 - The authentication method to use. An authentication method of *trust* disables password authentication. The use of *trust* is convenient for testing, but is **not recommended for production use**. Use `md5` or `scram-sha-256` for secure authentication in production.
+                 - trust
 
-    sudo yum install -y postgresql13 postgresql13-server
+        An example entry for use in a production environment, might be as follows:
 
-Before using the PostgreSQL server, you need to first initialize the database service using the command
+        .. code-block:: ini
 
-.. code-block:: console
+           host    all    all    0.0.0.0/0    md5
 
-    sudo /usr/pgsql-13/bin/postgresql-13-setup initdb
+        .. note::
 
-You can then proceed to start the database server as follows
+           Security may be further enhanced if SSL encryption is enforced on connections. This is done by specifying a connection type of *hostssl* rather than *host* in the configuration record. You must also enable SSL support in PostgreSQL, refer to the PostgreSQL documentation for settings required for this.
```

Comment:
Security can be further strengthened by enforcing SSL encryption on connections.